Open Judge System/Web/OJS.Web/Areas/Administration/Controllers/SubmissionsController.cs

```diff
@@ -206,12 +206,12 @@ public ActionResult Update(ModelType model)
 
                 if (this.ModelState.IsValid)
                 {
-                    var entity = this.GetById(model.Id) as DatabaseModelType;
-                    entity.Processed = false;
-                    entity.Processing = false;
+                    var submission = this.GetById(model.Id) as DatabaseModelType;
```

Comment:
Is this cast necessary? Why not simply use the `GenericRepository`'s `GetById`  method for retrieving a submission from the data storage?